contracts/ERC165_base.cairo

```diff
@@ -3,6 +3,8 @@
 from starkware.cairo.common.cairo_builtins import HashBuiltin, SignatureBuiltin
 from starkware.cairo.common.math import assert_not_equal
 
+const TRUE = 1
```

Comment:
what do you think about defining them in a `constants` module and just import it? cross-codebase redefinition seems weak

Reply:
Agreed. Good idea